packages/webapp/src/utils/acounts.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * Utilities to deal with burner accounts.
+ *
+ * @module utils/accounts
+ */
+
+import {
+    createPublicClient,
+    createWalletClient,
+    http,
+    getContract,
+    concat,
+    encodeFunctionData,
+} from 'viem'
+import { localhost } from 'viem/chains'
+import { generatePrivateKey, privateKeyToAccount } from 'viem/accounts'
+import {
+    simpleAccountFactoryAbi,
+    simpleAccountAbi,
+    paymasterAbi
+} from 'src/utils/abis'
+
+interface UserOperation {
+    sender: `0x${string}`
+    nonce: number
+    initCode: `0x${string}`
+    callData: `0x${string}`
+    callGasLimit: number
+    verificationGasLimit: number
+    preVerificationGas: number
+    maxFeePerGas: number
+    maxPriorityFeePerGas: number
+    paymasterAndData: `0x${string}`
+    signature: `0x${string}`
+}
+
+const publicClient = createPublicClient({
+    chain: localhost,
+    transport: http()
+})
+
+const paymasterAddress = process.env.PAYMASTER_ADDRESS as `0x${string}`
+const paymaster = getContract({
+    address: paymasterAddress,
+    abi: paymasterAbi,
+    publicClient
+})
+
+export async function createAccount() {
+
+    // use simple account factory to deploy a new smart account
+    // get the address of the smart account
+    // create sdk that reads private key from the browser storage and signs transactions
+
+    const privateKey = generatePrivateKey()
+    // store this private key in browser storage (is this secure?)
+    localStorage.setItem("privateKey", privateKey)
+
+    const account = privateKeyToAccount(privateKey)
+
+    // TODO: change chain
+    const walletClient = createWalletClient({
+        account,
+        chain: localhost,
+        transport: http()
+    })
```

Comment:
Same, `import { getWalletClient } from "wagmi/actions"`

Reply:
In there a way for us to pass in the `account` object in that case?

Reply:
I think we should setup so that whenever we use a burner key, we setup the wagmi wallet with it. So this would mean moving the account creation / insertion to `src/wagmi.ts` and gating it with an option (I guess for now just a file-level boolean is fine).

Reply:
You're right. I think we need to figure out a way to pass this client to wagmi however. i.e. when we use a burner key, we eschew the walletconnect prompting etc, and instead configure the global wagmi wallet to use the burner key.

Reply:
Maybe the move is to create a custom connector in wagmi. This is poorly documented. Maybe the MockConnector suffices? https://wagmi.sh/core/connectors/mock

I'll try having a look soon.

Reply:
Example for creating connectors: https://wagmi.sh/examples/custom-connector
Connector interface: https://github.com/wagmi-dev/wagmi/blob/main/packages/connectors/src/base.ts